content/en/account_management/saml/troubleshooting.md

```diff
@@ -0,0 +1,82 @@
+---
+title: SAML Troubleshooting
+kind: documentation
+description: Troubleshoot SAML issues for your Datadog account
+further_reading:
+- link: "https://www.samltool.com/online_tools.php"
+  tag: "Developer Tools"
+  text: "Identify your assertions with developer SAML tools"
+
+---
+
+## Overview
+
+This page provides troubleshooting instructions for common errors during Security Assertion Markup Language (SAML) authentication.
+
+## IdP metadata file errors
+
+If you are having trouble with updating your IdP metadata file, it is recommended to verify that the metadata file you are trying to upload is valid.
+
+To validate your metadata file:
+
+1. Choose a SAML validation tool, such as the [SAML developer tool][1] by OneLogin.
+2. Paste your metadata into the XML field and select **Metadata** in the XSD (schema file) field.
+3. Click **Validate XML With the XSD Schema**.
+
+## Roles errors
+
+When mappings are enabled, users logging in with SAML to a Datadog account are stripped of their current roles and reassigned to new roles based on the details in their SAML assertion passed on from your Identity Provider, and the mappings set within Mappings settings.
+
+Users who log in with SAML and do not have the values that map to a Datadog role are stripped of all roles and will not be allowed to log in.
+
+If you have group mappings set and are not able to see your roles, you group mappings in the Datadog application may appear differently in your IdP. To verify:
+
+1. Retrieve your IdP's SAML assertion for your account. Use browser tooling, such as [extensions][6], to retrieve your SAML assertion.
+2. Navigate to the Team page in the bottom left corner of Datadog.
+3. Select the **Mappings** tab.
+4. Compare the attributes provided by your IdP in your SAML assertion to the attributes set in the **Mappings** tab.
+5. Resolve any discrepencies in either the Datadog Mappings settings, or within your IdP settings.
+
+If you are having trouble logging in because of a role-based error, contact your Administrator to complete the troubleshooting steps above.
```

Comment:
Can we show what those errors look like? Here is an example: blob:https://datadoghq.atlassian.net/83e9cb85-c1c5-4e59-836e-c49e01680ed6#media-blob-url=true&id=100bd006-d99d-4520-bbb9-d6ed6dab9712&contextId=1248854720&collection=contentId-1248854720

Reply:
@vibrantlife I'm getting a 404 dead link error on this page?

![Screen Shot 2021-09-30 at 5 29 56 PM](https://user-images.githubusercontent.com/76412946/135548010-c7309e74-e014-4e14-9f0f-9e4d82d3f65a.png)

Reply:
Oops. Try this blob:https://datadoghq.atlassian.net/3b2f91cb-484f-468f-bee5-819feff7367f#media-blob-url=true&id=100bd006-d99d-4520-bbb9-d6ed6dab9712&contextId=1248854720&collection=contentId-1248854720

Reply:
I'm still getting the same 404 error 😅

![Screen Shot 2021-10-01 at 5 32 32 PM](https://user-images.githubusercontent.com/76412946/135698040-4e06b682-8106-42e4-9847-d0fa8aebad02.png)